roles/common/templates/rules.v4.j2

```diff
@@ -29,6 +29,10 @@ COMMIT
 :PREROUTING ACCEPT [0:0]
 :POSTROUTING ACCEPT [0:0]
 
+{% if wireguard_enabled and wireguard_port|int == 53 %}
+# Handle the special case of allowing access to WireGuard over port 53
```

Comment:
Why would not make port 53 also configurable? We can keep `wireguard_port: 51820` and the new variable `wireguard_port_alt: 53`, where `_alt` means an alternative port which is configured by iptables?

Reply:
I'm not sure I understand your suggestion. I didn't want to change the meaning of `wireguard_port`, or the code to generate client config files and to open provider firewall ports would also have to change. I also intended to make this change transparent to users.

I felt hardcoding `53` was OK because there are already several hardcoded port numbers in the `iptables` templates.

There's probably a better name for `wireguard_port_alt`, which is where WireGuard is told to *really* listen if the user happens to set `wireguard_port: 53` and thereby trigger this code. I was trying to keep it short.

If I was better at Jinja2 I might have tested `wireguard_port` against an array or list of numbers, not just `53`. For example, setting `wireguard_port` to `68` or `546` might also fail without this `iptables` trick, depending on the provider. Actually, this trick might not work for `68` on EC2.

Reply:
My point was about not hardcoding 53 port and make it configurable

Reply:
We could do that. I think it needs to be a new variable. Should it be in `config.cfg` or the WireGuard defaults where `_alt` is now?

Reply:
I changed to using a variable for the port. I think I'll also update the comments in `config.cfg` to mention possible issues with mobile data.